src/test/java/io/strimzi/kafka/bridge/http/ProducerTest.java

```diff
@@ -222,14 +238,16 @@ void sendBinaryMessageWithKey(VertxTestContext context) {
         });
     }
 
+    @Disabled("FLAKY somethi")
```

Comment:
what about this?

Reply:
I must investigate, why this test is flaky.

Reply:
It is caused  by PERIODIC_DELAY were set on 200ms,  and some messages were `inorder`. I have set that  PERIODIC_DELAY to 1s a now it's everything ok.